### PR TITLE
Add entity extraction capability

### PR DESCRIPTION
**Pull request by c-w**
The Cassandra schema contains a field for entities separate from places so this pull request ensures that we have the necessary data available to write the entity information to Cassandra.

### CODE REVIEW

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/PlaceRecognizer.scala [hunk 1]
```diff
@@ -1,62 +1,23 @@
...
-
 import com.microsoft.partnercatalyst.fortis.spark.logging.Loggable
-import com.microsoft.partnercatalyst.fortis.spark.transforms.ZipModelsProvider
+import com.microsoft.partnercatalyst.fortis.spark.transforms.entities.EntityRecognizer
 import com.microsoft.partnercatalyst.fortis.spark.transforms.nlp.OpeNER
-import ixa.kaflib.Entity
-
-import scala.collection.JavaConversions._
-import scala.util.{Failure, Success, Try}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.nlp.OpeNER.entityIsPlace
```
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/PlaceRecognizer.scala]: nit: In this case, I think qualifying the name inline would make it more obvious to the reader that this helper is part of OpeNER.

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/people/PeopleRecognizer.scala [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+class PeopleRecognizer(
+  modelsSource: Option[String] = None,
+  enabledLanguages: Set[String] = OpeNER.EnabledLanguages
+) extends Serializable with Loggable {
+
+  @volatile private lazy val entityRecognizer = createEntityRecognizer()
+
+  def extractPeople(text: String, language: String): List[Tag] = {
+    entityRecognizer.extractEntities(text, language).filter(entityIsPerson)
+      .map(entity => Tag(name = entity.getStr, confidence = 1.0))
```
Comment (kevinhartman) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/people/PeopleRecognizer.scala]: For my own understanding, why are we using tags for people but not place entities?

**Review by kevinhartman — APPROVED** [review]
LGTM

**Review by jcjimenez — APPROVED** [review]
LGTM

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/PlaceRecognizer.scala (hunk 1 above)
Comment (c-w) [reply src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/PlaceRecognizer.scala]: Done in 2934106.

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/people/PeopleRecognizer.scala (hunk 2 above)
Comment (c-w) [reply src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/people/PeopleRecognizer.scala]: Yeah, not happy with that inconsistency too. Fixed in e0daddc.

**Pull request merged**